spec.html

```diff
@@ -12620,12 +12623,12 @@ <h1>
             1. If _extensible_ is *false*, return *false*.
             1. If _O_ is *undefined*, return *true*.
             1. If ! IsAccessorDescriptor(_Desc_) is *true*, then
-              1. Create an own accessor property named _P_ of object _O_ whose [[Get]], [[Set]], [[Enumerable]], and [[Configurable]] attribute values are described by _Desc_. If the value of an attribute field of _Desc_ is absent, the attribute of the newly created property is set to its <emu-xref href="#table-object-property-attributes">default value</emu-xref>.
+              1. Create an own accessor property named _P_ of object _O_ whose [[Get]], [[Set]], [[Enumerable]], and [[Configurable]] attributes are set to the value of the corresponding field in _Desc_ if it has the field, or to the attribute's <emu-xref href="#table-object-property-attributes">default value</emu-xref> otherwise.
```

Comment:
```suggestion
              1. Create an own accessor property named _P_ of object _O_ whose [[Get]], [[Set]], [[Enumerable]], and [[Configurable]] attributes are set to the value of the corresponding field in _Desc_ if _Desc_ has that field, or to the attribute's <emu-xref href="#table-object-property-attributes">default value</emu-xref> otherwise.
```

Just to eliminate a possible ambiguity.

Reply:
Which possible ambiguity? The only thing that can have a field is a record, and the only record mentioned in the step is `_Desc_`, so I think it's fairly clear that the "it" in "it has the field" must refer to `_Desc_`.

Reply:
That's true on a very careful reading, but on a casual reading "it" could be taken to refer to the property, and the reader has to figure out that this would be incoherent and so must not be the intended reading. I prefer to be explicit.